scan-engine/src/test/java/de/tillhub/scanengine/data/ScannerManufacturerTest.kt

```diff
@@ -0,0 +1,20 @@
+package de.tillhub.scanengine.data
+
+import br.com.colman.kotest.android.extensions.robolectric.RobolectricTest
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.matchers.shouldBe
+import org.robolectric.shadows.ShadowBuild
+
+@RobolectricTest
+internal class ScannerManufacturerTest : FunSpec({
+
+    test("SUNMI device") {
+        ShadowBuild.setManufacturer("SUNMI")
+        ScannerManufacturer.get() shouldBe ScannerManufacturer.SUNMI
+    }
+
+    test("OTHER device") {
+        ShadowBuild.setManufacturer("OTHER")
```

Comment:
based on the code we want to test with something other than sunmi, without context I'd assume that "OTHER" has a proper mapping. 
I am suggesting to change "OTHER" to "NOT_SUNMI"